doc/modules/classes.rst

```diff
@@ -1357,6 +1357,28 @@ Low-level methods
    utils.shuffle
 
 
+:mod:`sklearn.plot`: Plotting functions
+=======================================
+
+.. automodule:: sklearn.plot
+   :no-members:
+   :no-inherited-members:
+
+This module is experimental. Use at your own risk.
+Use of this module requires the matplotlib library,
+version 1.5 or later (preferably 2.0).
```

Comment:
yea I meant to say 2 is better in general. but it looks like a preference of our API so i'll remove it.